auslib/AUS.py

```diff
@@ -52,9 +52,23 @@ def __init__(self):
         self.rand = functools.partial(randint, 0, 99)
         self.log = logging.getLogger(self.__class__.__name__)
 
+    def get_emergency_shutoff(self, product, channel):
+        where = dict(product=product, channel=channel)
+        emergency_shutoffs = dbo.emergencyShutoff.getEmergencyShutoffs(where=where)
+        return emergency_shutoffs[0] if emergency_shutoffs else None
+
     def evaluateRules(self, updateQuery):
         self.log.debug("Looking for rules that apply to:")
         self.log.debug(updateQuery)
+
+        if 'product' in updateQuery and 'channel' in updateQuery:
```

Comment:
I think it's safe to assume that product and channel are in the update query, since they exist in all update URL styles.